org-noter/org-noter
# Feature/generalize file opening process

## Problem

I'd like to be able to modify the notes opening process. Right now it's assumed that only `Notes.org` file is to be used within a certain number of available directories.

I'd like to be able to override the process so that I can stores notes  files within my org-roam.

## Solution


Similar to other refactorings:

1. add a hook to be called when looking for a notes file
2. by default set the hook to call `org-noter--create-session-from-publication-file-default`. This is the original implementation of the notes file opening process.

## Checklist

- [ ] I added unit tests. (covered by original unit tests)

## Steps to Test

This should be a completely transparent change:

1. Open a PDF file
2. `M-x org-noter`

Org-noter should launch.
